legend-engine-xts-relationalStore/legend-engine-xt-relationalStore-generation/legend-engine-xt-relationalStore-pure/src/main/resources/core_relational/relational/relationalMappingExecution.pure

```diff
@@ -42,6 +42,10 @@ import meta::relational::milestoning::*;
 import meta::pure::router::routing::*;
 import meta::pure::router::store::metamodel::*;
 
+Class meta::relational::mapping::DomainTypeToSourceDataTypePair extends Pair<String, meta::relational::metamodel::datatype::DataType>
```

Comment:
we don't need this, right?

Reply:
it's used in the getSourceDataTypeFromType to create the Pairs. I followed the same pattern as in the pureToSqlQuery where it maps the function expression to a method